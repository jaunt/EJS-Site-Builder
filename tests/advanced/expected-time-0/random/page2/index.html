<html lang="en">

<head>
  <title>2</title>
</head>

<body>
  Alternate:
  <i> <div>hello 2</div> </i>
</body>

</html>