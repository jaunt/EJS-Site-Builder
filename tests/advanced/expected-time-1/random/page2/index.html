<html lang="en">

<head>
  <title>2</title>
</head>

<body>
  Alternate wrapper:
  <i> <div>hello 2</div> </i>
</body>

</html>