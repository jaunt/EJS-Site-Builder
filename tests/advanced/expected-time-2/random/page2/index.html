<html lang="en">

<head>
  <title>2</title>
</head>

<div class="but">about me</div>
<div>My buttons are clicky at times</div>

grape,apple
Fruit: grape

grape,apple
Fruit: apple


<body>
  Alternate:
  <i> <div>s hello 22</div> </i>
</body>

</html>