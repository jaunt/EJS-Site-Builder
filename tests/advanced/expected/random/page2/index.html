<html lang="en">

<head>
  <title>2</title>
</head>

<body>
  Alternate:
  <i> <div>1637038322874</div> </i>
</body>

</html>